scripts/package

```diff
@@ -64,11 +64,11 @@ fi
 
 TAG=$TAG REPO=${REPO} go run ../pkg/image/export/main.go build/system-charts build/charts $IMAGE $AGENT_IMAGE
 
-if [ "$ARCH" = "amd64" ]; then
+if [ ${ARCH} == amd64 ]; then
```

Comment:
Curious if this is a fix or just a best practice

Reply:
Alignment with what the script is doing above.